### ISSUE TITLE
Missing image resource in canvas

### ISSUE THREAD

**Issue by agentg2007** [microsoft/PowerApps-Samples#133]
Hi,

I created an test app that has a search input with search icon in the button.
In sandbox, everyting works fine and I can see my imgs/rearch.png.

![image](https://user-images.githubusercontent.com/43179883/85011969-e5fc1300-b194-11ea-935f-d707dda98c18.png)

But when I applied it in a PowerApp Canvas, I got this.

![image](https://user-images.githubusercontent.com/43179883/85012397-894d2800-b195-11ea-8143-ebb7874206d3.png)

The image is now missing. I only got the bundle and the css.

I also checked the published manifest and it's all there.

What did I done wrong?

**Comment by agentg2007** [comment]
I confirmed it's a bug. I build the sample TS_ImageUploadControl and uploaded the solution. And I executed this code `context.resources.getResource(DefaultImageFileName, data => console.log("ImageData", data), () => console.warn("Unable to get data", context.resources))`

I got this:
![image](https://user-images.githubusercontent.com/43179883/85057296-2a0e0880-b1d3-11ea-8871-189ab005bca4.png)

Same behavoiur, image is not added in the resource.
